app/controllers/problems_controller.rb

```diff
@@ -61,7 +61,7 @@ def show
   end
 
   def create_issue
-    body = render_to_string "issue_trackers/issue", layout: false, formats: [:txt]
+    body = render_to_string "issue_trackers/issue", layout: false, formats: [:md]
```

Comment:
Sounds good. I'm thinking we could even potentially leave that decision to Issue#save, where we have everything we need to make that happen (the issue tracker object and the problem object). But we can probably solve this as part of https://github.com/errbit/errbit/issues/783

I'll ping you from that one and we can decide what to do.